### DISCUSSION TITLE
AOSP 11.0 v309

### DISCUSSION THREAD

**Discussion by phhusson** [phhusson/treble_experimentations#1889] [General]
- June security patch level
- [Zenfone 8] Fix boot, audio, camera hole, dt2w
- [Doogee S95] Add support for dt2w (always enabled, not togglable)
- Fix BT voice calls on some Exynos devices
- Expand "use alternate audio policy" to Exynos devices. Used to fix for instance voice BT voice calls
- Expand "use alternate audio policy" to newer Qualcomm devices. Can fix jack for some
- Fix voice calls no longer working after some time (Thanks @eremitein)

Treble Presets:
- Realme C12: Fix headset detection and backlight control (Thanks @HemanthJabalpuri)

<hr /><em>This discussion was created from the release <a href='https://github.com/phhusson/treble_experimentations/releases/tag/v309'>AOSP 11.0 v309</a>.</em>

**Comment by linked380** [comment]
Thank You !

**Comment by Osanosa** [comment]
Could you fix WiFi Hotspot in next release on BV6800Pro?
(Issue reported but noone replayed)

**Comment by lostmsu** [comment]
Does this include June security patch?

**Reply by eremitein** [reply]
yes

**Reply by phhusson** [reply]
Good point. Added to the changelog.

**Reply by turuncu2011** [reply]
After installing AOSP 11.0 v 309, status bar and navigation bar disappeared. This bug needs to be fixed.

**Reply by bukyapavankumar** [reply]
> Good point. Added to the changelog.

Magisk support bringing back wen?

**Comment by jepoy313** [comment]
How to fix crashing auto force close when playing games like call of duty mobile

**Comment by Hausemaster8281** [comment]
Hello everyone, till the v308 build everything worked fine, on this build however the system ui doesn't start at all, I.e there's no quick settings, no navigation bar, no lock screen, etc. The first time it drops me of at the home screen and that's it, I can't do much, since of I open an app, I can't close it anymore due to the absence of navigation keys...
My architecture is arm and my device is a-only, is this issue device specific or, is it a bug in the latest builds, since both vanilla and gogapps builds give the same result...
Also if is a bug or not a bug is there a workaround for this so that the device can be used until the next release...
Edit: on the second boot the device gets stuck on a black screen, which is the lockscreen that hasn't loaded...

**Comment by klimentin40** [comment]
Hi, my P10 Lite supports Treble but when I flash any GSI and go to sound settings there is no Ringtone, Notification Sound and Alarm Sound, I have to manually add a Ringtone, Notification Sound and Alarm Sound.
Is there any way to fix this?

**Reply by rvlphee** [reply]
this is on every phone, because by default nothing should be there

**Reply by klimentin40** [reply]
> this is on every phone, because by default nothing should be there

Well that's not quite true, on my Xiaomi Redmi 6A all the sounds (Ringtone, Notification Sound and Alarm Sound) are there.

**Comment by ghost** [comment]
Can you provide Volte

**Reply by JRock37** [reply]
You can use Volte by clicking settings. Then, phh treble settings. From there click on IMS features. Check request IMS network and force 4g. Then click install IMS apk. Once it downloads install and reboot. Once rebooted go back to phh treble settings under IMS and click create IMS APN. Reboot again and you VOLTE settings will be in settings mobile network. If you click on mobile network you will now be able to see that it is enabled.

**Comment by turuncu2011** [comment]
After installing AOSP 11.0 v 309, status bar and navigation bar disappeared. This bug needs to be fixed.

**Reply by Hausemaster8281** [reply]
I also have the same bug
I opened both an issue and a discussion

**Comment by MaiklGureev** [comment]
Hello. thanks for the firmware.

**Comment by SteQuOFFICIAL** [comment]
Can anyone verify that this system can run on Google Pixel XL? Is there EXACT information about this software's compatibility somewhere?

**Reply by JRock37** [reply]
Use Treble check to verify. Then download the corresponding GSI. Either A, B, or AB version.

**Comment by 31n3r** [comment]
![6f34f8ac-b159-4a50-b657-ef96102fa371](https://user-images.githubusercontent.com/86453042/123364714-edda5a00-d542-11eb-96af-8656156dedad.jpg)
Hello, someone is using it in Alcatel 1S 2019?

**Reply by JRock37** [reply]
If you download and run treble check and it says you are compatible you are good to go. It will tell you if your device is A, B, or AB compatible. Then, just download the corresponding GSI.

**Reply by 31n3r** [reply]
The device is also known as TCL 5024A and supports Project Treble, but has a system partition of only A. This device is powered by a Unisoc SC9863A chipset coupled with 3GB of RAM and 32GB of internal storage. The device came out of the box with Android 9.0 Pie and backed by a 3060 mAh battery (Non-removable).

**Reply by JRock37** [reply]
> The device is also known as TCL 5024A and supports Project Treble, but has a system partition of only A. This device is powered by a Unisoc SC9863A chipset coupled with 3GB of RAM and 32GB of internal storage. The device came out of the box with Android 9.0 Pie and backed by a 3060 mAh battery (Non-removable).

Okay, you just need to flash the GSI with partition A only. I recommend trying this GSI as it as praised as being the most compatible. If you can get this one to boot then you can branch out and try others. I do not think you will have any problems since the device has been out for a while. As I mentioned below I believe the problem I am having is due to the modem being a 5g modem.

**Reply by 31n3r** [reply]
![Screenshot_20210626-170626](https://user-images.githubusercontent.com/86453042/123525766-285a0900-d6a1-11eb-9118-b5093d4da9c7.png)

Ok, I'm going to ask you a very silly question: which of these files is the ROM you tell me? 😅 is that I am new here on github

**Reply by JRock37** [reply]
Run treble check from google play store.

**Comment by JRock37** [comment]
I have a device that I received from TCL during a beta test. The device has not been officially released yet as the beta just ended. The device is fully treble AB compatible. It has Qualcomm 750g SOC with 5g radio. I flashed this GSI and so far everything seams to be working correctly.  However, the device randomly crashes due to the modem. It goes to a black screen that says Qualcomm crash dump. Can you please resolve this issue or if it can be resolved on my end tell me how to do so? Thank you for working to make such a highly compatible GSI. Keep up the good work 😉

**Reply by 31n3r** [reply]
the device I mentioned in the comment above 👆 is also made by TCL

**Reply by JRock37** [reply]
Mine has a problem with the radio used for mobile data. I think it is just a bug because my device is 5g. Since 5g is still a newer thing I think it just needs worked out. I do not think you will have this problem with yours.

**Reply by JRock37** [reply]
My issue with modem crashing turned out to be a bug with APN. If I so much as click on APN I lose the ability to use 5g. After setting to 5g I have to set singal measurement to RSSI or I will get the modem crash.

**Reply by JRock37** [reply]
Well, I thought I resolved the issue but I did not. 5g was working when I set the phone up at my house. I left to go to the gas station. I pulled the phone out of my pocket and it was unresponsive. It was stuck on a solid black screen with nothing on it. I held down the power button and restarted the phone. After restarting I am know stuck with Edge connection instead of 5g. I have included a logcat and radio file for you to review. Both can be downloaded from here https://filebin.net/393dgytnllray3lw
Man I just want to get one GSI running correctly =(

**Comment by JRock37** [comment]
When I install the gapps version for a/b I get the device is not google certified repeatedly. I have used device id to get my id and register it. However, this does not seam to work. Am I doing something wrong? I managed to get everything else working on my device which is a beta unit from TCL. Since it is a beta unit no recovery image is available for me to port twrp. Is it possible to flash the same was as flashing gsi using fastbootd?  If neither of these two options work can you please upload a non-rooted version with gapps for a/b 64bit since we have no way to remove root on a/b?

**Reply by JRock37** [reply]
Floss works fine but I cannot sync google contacts. Is there a way to permanently remove apps I do not want?

**Comment by Mahamadou1973** [comment]
Bjr. Est-ce que magisk fonctionne sur cette version ? Si oui, comment procéder à son installation? Sur les ancienne version, j'ai essayé en vain.

**Comment by Raullcito** [comment]
Oi, não estou conseguindo baixar nenhuma rom aosp, na metade do download o navegador diz: "o download requer autorização".
